sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -696,19 +693,17 @@ public String getVaultUrl() {
     }
 
     /**
-     * List secrets in the key vault. The list Secrets operation is applicable to the entire vault. The individual
-     * secret response in the flux is represented by {@link SecretProperties} as only the secret identifier and its
-     * attributes are provided in the response. The secret values and individual secret versions are not listed in the
-     * response. This operation requires the {@code secrets/list} permission.
-     *
-     * <p><strong>Code Samples</strong></p>
-     * <p>It is possible to get full Secrets with values from this information. Convert the {@link Flux} containing
-     * {@link SecretProperties secret properties} to
-     * {@link Flux} containing {@link KeyVaultSecret secret} using {@link SecretAsyncClient#getSecret(String, String)}
-     * within {@link Flux#flatMap(Function)}.</p>
+     * Lists secrets in the key vault. Each {@link SecretProperties secret} returned only has its identifier and
+     * attributes populated. The secret values and their versions are not listed in the response.
+     * This operation requires the {@code secrets/list} permission.
+     *
+     * <p><strong>Code sample</strong></p>
+     * <p>The sample below fetches the all the secret properties in the vault. For each secret retrieved, makes a call
+     * to {@link #getSecret(String, String) getSecret()} to get its value, and then prints it out.</p>
```

Comment:
In secret client, 
the link change is casted to getSecret(String, String).
Did you intend to keep it same here ?

Reply:
I had updated it because I thought it was too verbose showing "SecretyAsyncClient.getSecret(String, String)" the link would have taken them to the same place. I have no problems changing it to getSecret(String, String)

Reply:
okay, yeah let's keep it getSecret(String, String), to avoid user confusing it with getSecret() no parameter overload (which doesn't exist), in case they don't click on it.
Thanks.

Reply:
Fixed